Description: turn the y2k20 into a y2k40 problem, or fix test failures starting in 2020
 t/timelocal.t fails with
	not ok 31 - timelocal year for 1970 1 2 0 0 0
	#   Failed test 'timelocal year for 1970 1 2 0 0 0'
	#   at t/timelocal.t line 36.
	#          got: '170'
	#     expected: '70'
	not ok 37 - timegm year for 1970 1 2 0 0 0
	#   Failed test 'timegm year for 1970 1 2 0 0 0'
	#   at t/timelocal.t line 49.
	#          got: '170'
	#     expected: '70'
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2020-01-03

--- a/t/timelocal.t
+++ b/t/timelocal.t
@@ -25,7 +25,7 @@
 
 	# Test timelocal()
 	{
-		my $year_in = $year < 70 ? $year + 1900 : $year;
+		my $year_in = $year < 90 ? $year + 1900 : $year;
 		my $time = &timelocal($sec,$min,$hour,$mday,$mon,$year_in);
 		my($s,$m,$h,$D,$M,$Y) = localtime($time);
 		is $s, $sec, "timelocal second for @$_";
@@ -38,7 +38,7 @@
 
 	# Test timegm()
 	{
-		my $year_in = $year < 70 ? $year + 1900 : $year;
+		my $year_in = $year < 90 ? $year + 1900 : $year;
 		my $time = &timegm($sec,$min,$hour,$mday,$mon,$year_in);
 		my($s,$m,$h,$D,$M,$Y) = gmtime($time);
 		is $s, $sec, "timegm second for @$_";
